ale_linters/go/gometalinter.vim

```diff
@@ -6,7 +6,7 @@ if !exists('g:ale_go_gometalinter_options')
 endif
 
 function! ale_linters#go#gometalinter#GetCommand(buffer) abort
-    return 'gometalinter '
+    return 'gometalinter --include=''^' . expand('%:p') . '.*$'' '
```

Comment:
Yeah indeed, but I doubt this will cause issues... I used the exact same code in `vim-go` quite some time ago and until now never got a complaint.

And since your not a Go user... FYI Vim in combination with `vim-go` is probably the most used setup for Go developers:

![image](https://cloud.githubusercontent.com/assets/4171547/26275477/b3b02ae4-3d61-11e7-9c03-ecbdb960e6c1.png)
https://blog.golang.org/survey2016-results

So I think we're good, but of course ping me if an related issue does come in. Thanks 👍